Britannia/events/religious_decision_events.txt

```diff
@@ -0,0 +1,2878 @@
+	}
+
+	option = {
+		name = religious_decision.0101.dakshinamoorthy
+		trigger = {
+			OR = {
+				faith = faith:shaivism
+				bhakti_foundational_faith_trigger = { FAITH = faith:shaivism }
+			}
+			NOT = { has_character_modifier = bhakti_shaivism_dakshinamoorthy }
+		}
+		set_bhakti_effect = { BHAKTI = bhakti_shaivism_dakshinamoorthy }
+	}
+
+	# Smartism
+	option = {
+		name = religious_decision.0101.ganesha
+		trigger = {
+			OR = {
+				faith = faith:smartism
+				bhakti_foundational_faith_trigger = { FAITH = faith:smartism }
+			}
+			NOT = { has_character_modifier = bhakti_smartism_ganesha }
+		}
+		set_bhakti_effect = { BHAKTI = bhakti_smartism_ganesha }
+	}
+
+	option = {
+		name = religious_decision.0101.vishnu
+		trigger = {
+			OR = {
+				faith = faith:smartism
+				bhakti_foundational_faith_trigger = { FAITH = faith:smartism }
+			}
+			NOT = { has_character_modifier = bhakti_smartism_vishnu }
+		}
+		set_bhakti_effect = { BHAKTI = bhakti_smartism_vishnu }
+	}
+
+	option = {
+		name = religious_decision.0101.shiva
+		trigger = {
+			OR = {
+				faith = faith:smartism
+				bhakti_foundational_faith_trigger = { FAITH = faith:smartism }
+			}
+			NOT = { has_character_modifier = bhakti_smartism_shiva }
+		}
+		set_bhakti_effect = { BHAKTI = bhakti_smartism_shiva }
+	}
+
+	# Opt-Out
+	option = {
+		name = religious_decision.0101.optout
+		flavor = religious_decision.0101.flavor
+
+		ai_chance = {
+			base = 0
+		}
+	}
+}
+
+# MODIFIED BY BTWK #
+religious_decision.0102 = {
+	type = character_event
+	title = religious_decision.0102.t
+	desc = religious_decision.0102.desc
+	theme = faith
+	left_portrait = {
+		character = root
+		animation = personality_rational
+	}
+
+	# # Odin
+	# option = {
+	# 	name = religious_decision.0102.odin
+	# 	trigger = {
+	# 		NOT = { has_character_modifier = bhakti_germanic_generic_odin }
+	# 	}
+	# 	set_bhakti_effect = { BHAKTI = bhakti_germanic_generic_odin }
+	# }
+
+	# # Ullr
+	# option = {
+	# 	name = religious_decision.0102.ullr
+	# 	trigger = {
+	# 		is_vaguely_danish_bhakti_trigger = no
+	# 		NOT = { has_character_modifier = bhakti_germanic_not_danish_ullr }
+	# 	}
+	# 	set_bhakti_effect = { BHAKTI = bhakti_germanic_not_danish_ullr }
+	# }
+
+	# # Tyr
+	# option = {
+	# 	name = religious_decision.0102.tyr
+	# 	trigger = {
+	# 		is_vaguely_danish_bhakti_trigger = yes
+	# 		NOT = { has_character_modifier = bhakti_germanic_danish_tyr }
+	# 	}
+	# 	set_bhakti_effect = { BHAKTI = bhakti_germanic_danish_tyr }
+	# }
+
+	# # Thor
+	# option = {
+	# 	name = religious_decision.0102.thor
+	# 	trigger = {
+	# 		NOT = { has_character_modifier = bhakti_germanic_generic_thor }
+	# 	}
+	# 	set_bhakti_effect = { BHAKTI = bhakti_germanic_generic_thor }
+	# }
+
+	# # Freyr
+	# option = {
+	# 	name = religious_decision.0102.freyr
+	# 	trigger = {
+	# 		NOT = { has_character_modifier = bhakti_germanic_generic_freyr }
+	# 	}
+	# 	set_bhakti_effect = { BHAKTI = bhakti_germanic_generic_freyr }
+	# }
+
+	# NORSE
```

Comment:
May be easier to combine the different faith variants of the same god into a single option and just use if/else blocks... up to you though